atest/acceptance/multiple_browsers_multiple_windows.robot

```diff
@@ -0,0 +1,147 @@
+*** Setting ***
+Documentation     These tests must open own browser because windows opened by
+...               earlier tests would otherwise be visible to Get Window XXX keywords
+...               even if those windows were closed.
+Suite Setup        Open 3 Browsers with Windows
+Suite Teardown     Close All Browsers
+Resource          resource.robot
+
+*** Variables ***
+@{BrowserA_exp_Titles}=    WindowA1    WindowA2    WindowA3
```

Comment:
changed them to capital. 
But, i let "BrowserA" at This case because it is also the alias and used multiple times. So it is more a Name that a variable?! 
like BrowserA_EXP_TITLES